### PR TITLE
i18n: (zh-CN) Add translations for #5746 & #5750

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ykzts** [comment]
cc @Artoria2e5 @KenOokamiHoro @leopku @miuchan @vidarlee

**Comment by SerCom-KC** [comment]
In that case please also review #5725 & #5764, these two are not thoroughly checked yet

File: app/javascript/mastodon/locales/zh-CN.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "account.follows_you": "关注了你",
   "account.media": "媒体",
   "account.mention": "提及 @{name}",
-  "account.moved_to": "{name} has moved to:",
+  "account.moved_to": "{name} 已经迁移到：",
```
Comment (Artoria2e5) [inline app/javascript/mastodon/locales/zh-CN.json]: Wonder why my brain keeps telling me 迁移 is something that comes with 被. Time to fix this brain I guess…

**Review by Artoria2e5 — APPROVED** [review]

**Review by ykzts — APPROVED** [review]

File: app/javascript/mastodon/locales/zh-CN.json (hunk 1 above)
Comment (SerCom-KC) [reply app/javascript/mastodon/locales/zh-CN.json]: lol

**Pull request merged**